Makefile

```diff
@@ -1,9 +1,9 @@
-VENV_DIR ?= .venv
```

Comment:
What's with the Makefile changes?

Reply:
It was broken, so I fixed it so I could use it to run the tests

Reply:
Ah, I see. Feels like it belongs a separate commit in that case.

Reply:
reverted

Reply:
:+1: didn't necessarily mean a separate PR btw. Drive by fixes are great, but in my experience it pays off over time to keep commits logically atomic (as far as reasonably possible)